frame/oracle/src/tests.rs

```diff
@@ -856,6 +856,70 @@ fn submit_price_fails_stake_less_than_asset_slash() {
 	});
 }
 
+fn halborm_test_price_manipulation() {
```

Comment:
Should we also cover this test case with integration tests, to be safe?

Reply:
If it is not too hard in TS then sure I can try.

Reply:
Shouldn't be that hard, though if there's too much on your plate currently, the QA team can take care of this too. :)

Reply:
if QA team can do this then it will be great , I am working on other issues for halborn issue.